gslib/tests/test_cp.py

```diff
@@ -3325,6 +3325,22 @@ def test_kms_key_correctly_applied_to_composite_upload(self):
     with SetBotoConfigForTest([('GSUtil', 'prefer_api', 'json')]):
       self.AssertObjectUsesCMEK(obj_suri, key_fqn)
 
+  @SkipForS3('No composite upload support for S3.')
+  def test_nearline_applied_to_parallel_composite_upload(self):
+    bucket_uri = self.CreateBucket()
```

Comment:
To emphasize the class difference, can you set the default class to `standard` here?

Reply:
Done.